cleverhans/utils_tf.py

```diff
@@ -17,30 +17,6 @@
 _logger = create_logger("cleverhans.utils.tf")
 
 
-def model_loss(y, model, mean=True):
```

Comment:
Can you add a deprecation warning and call the new function you defined in loss.py to make this change backward compatible before we remove this function in 6 months?

Reply:
Yes.